components/src/core/components/SchemaForm/SchemaForm.vue

```diff
@@ -195,6 +199,9 @@ export default defineComponent({
           layoutChildObj[slotName]
             .map((field) => {
               if (field.visible !== false && field.type === 'button') {
+                if(isDisabled.value){
+                  return null;
```

Comment:
this will prevent the form buttons being rendered? shouldn't we instead disable them?

Reply:
@Super-Chama I have checked this with Mike. If the entire form is disabled then the action buttons shouldn't be rendered as a pattern. That is why it is not rendered. I can understand the reason you mentioned considering the semantics.

Reply:
@Super-Chama It has changed as you mentioned. Since action buttons can be anything disabling all will not work. People have to consider in their form logic whether to show or not buttons